crates/fuel-core/src/graphql_api/database.rs

```diff
@@ -134,7 +141,7 @@ impl ReadView {
     pub fn transaction(&self, tx_id: &TxId) -> StorageResult<Transaction> {
         let result = self.on_chain.transaction(tx_id);
         if result.is_not_found() {
-            if let Some(tx) = self.old_transaction(tx_id)? {
+            if let Some(tx) = self.off_chain.old_transaction(tx_id)? {
```

Comment:
I prefer the old helper... although the meaning of "old" here isn't clear to me.

Reply:
<img width="644" alt="Screenshot 2024-10-14 at 2 31 18 PM" src="https://github.com/user-attachments/assets/c347861d-cac5-4f9f-808e-43864987acb6">